packages/yew-functional-macro/tests/function_attr/generic-props-fail.rs

```diff
@@ -0,0 +1,20 @@
+use yew::prelude::*;
+use yew_functional::function_component;
+
+#[derive(Clone, Properties, PartialEq)]
+struct Props {
+    a: usize,
+}
+
+#[function_component(Comp)]
+fn comp<P: Properties + PartialEq>(_props: &P) -> Html {
+    html! {
+        <p></p>
+    }
+}
+
+fn compile_pass() {
+    html! { <Comp<Props> /> }; // missing prop 'a'
+}

```

Comment:
I would like to add the following cases as well, just for posterity. Also note the function name change.

```suggestion
fn compile_fail() {
    // missing prop 'a'
    html! { <Comp<Props> /> };
    
    // invalid type parameter
    html! { <Comp<INVALID> /> };
    // parameter doesn't match bounds
    html! { <Comp<()> /> };

    // missing type param
    html! { <Comp /> };
}

```

Reply:
Oops, should have paid more attention to the function name :)